catalog.html

```diff
@@ -160,33 +160,41 @@ <h2 class="sorting-heading">Сортировка:</h2>
               <div class="catalog-product-cards">
                 <ul class="catalog-product-cards-list">
                   <li class="catalog-product-cards-item">
-                    <img class="catalog-product-image" src="images/selfie-stick/selfie-stick-1.jpg" alt="Любительская селфи палка.">
-                    <div class="catalog-product-description">
-                      <b class="catalog-product-name">Любительская селфи палка</b>
-                      <span class="catalog-product-price">500 р</span>
-                    </div>
-                  </li>
+                    <a class="catalog-product-cards-link" href="#">
+                      <img class="catalog-product-image" src="images/selfie-stick/selfie-stick-1.jpg" alt="Любительская селфи палка.">
+                      <div class="catalog-product-description">
+                        <b class="catalog-product-name">Любительская селфи палка</b>
+                        <span class="catalog-product-price">500 р</span>
+                      </div>
+                    </a>
 
+                  </li>
                   <li class="catalog-product-cards-item">
-                    <img class="catalog-product-image" src="images/selfie-stick/selfie-stick-2.jpg" alt="Профессиональная селфи-палка.">
-                    <div class="catalog-product-description">
-                      <b class="catalog-product-name">Профессиональная селфи-палка</b>
-                      <span class="catalog-product-price">1500 р</span>
-                    </div>
+                    <a class="catalog-product-cards-link" href="#">
+                      <img class="catalog-product-image" src="images/selfie-stick/selfie-stick-2.jpg" alt="Профессиональная селфи-палка.">
+                      <div class="catalog-product-description">
+                        <b class="catalog-product-name">Профессиональная селфи-палка</b>
+                        <span class="catalog-product-price">1500 р</span>
```

Comment:
```suggestion
                        <span class="catalog-product-price">1500 ₽</span>
```
Можно скопировать символ из макета